### ISSUE TITLE
Kube Events

### ISSUE THREAD

**Issue by danielepolencic** [rootsongjc/awesome-cloud-native#59]
I'd like to include [Kube Events](https://kube.events) to the list under the "Community" section.

If you deem this not relevant, I'm happy for this issue to be closed.

**Comment by rootsongjc** [comment]
@danielepolencic I don't think it is a community or organization, it looks like a collection of Kubernetes Events. Especially, I don't see any introduction about this "community".
